README.md

```diff
@@ -78,6 +78,20 @@ mergeImages(['/body.png', '/eyes.png', '/mouth.png'], {
 
 Which will look like this:
 
+### Opactiy
+
+Set the opacity of an image.
+
+```js
+mergeImages([
+  { src: 'body.png', x: 0, y: 0 },
+  { src: 'eyes.png', x: 32, y: 0, opacity: 0.5 }
+])
+  .then(b64 => ...);
+  // data:image/png;base64,iVBORw0KGgoAA...
+```
+
+
```

Comment:
Can you add a demo image of what the output will look like, like the other examples?

Reply:
okay... I will do this tomorrow :)

Reply:
resolved